sql/2024/fonts/development/fonts_compiler.sql

```diff
@@ -0,0 +1,48 @@
+-- Section: Design
+-- Question: Which compilers are used?
+
+CREATE TEMPORARY FUNCTION COMPILER(version STRING) AS (
+  CASE
+    WHEN REGEXP_CONTAINS(version, r'(?i)(Core \d|PS \d|hotconv|makeotf)') THEN 'Adobe Font Development Kit for OpenType'
+    WHEN REGEXP_CONTAINS(version, r'(?i)FontCreator') THEN 'FontCreator'
+    WHEN REGEXP_CONTAINS(version, r'(?i)Fontself') THEN 'Fontself Maker'
+    WHEN REGEXP_CONTAINS(version, r'(?i)ttfautohint') THEN 'FreeType'
+    WHEN REGEXP_CONTAINS(version, r'(?i)(FEAKit|Glyphs)') THEN 'Glyphs.app'
+    WHEN REGEXP_CONTAINS(version, r'(?i)gftools') THEN 'Google Fonts Tools'
```

Comment:
Maybe this should be fontmake?

Reply:
Renamed. Renamed also “Adobe Font Development Kit” to simply “makeotf.”